docs/TASKS.md

```diff
@@ -1,11 +1,31 @@
 Tasks
 =====
 
-A task describes the how, what, and when to do for a Snap job.  A task is described in a task _manifest_, which can be either JSON or YAML<sup>1</sup>.
+A task describes the how, what, and when to do for a Snap job.  A task is described in a task _manifest_, which can be either JSON or YAML<sup>1</sup>. The manifest can be divided into two parts: Header and Workflow.
```

Comment:
I made some changes to the layout, moved the tldr back to the top, and updated the state diagram. Let me know what you think of these changes.